src/java/org/apache/cassandra/exceptions/InvalidColumnTypeException.java

```diff
@@ -54,7 +54,7 @@ private static String msg(ByteBuffer name,
                               String reason)
     {
         return String.format("Invalid type %s for column %s: %s",
-                             invalidType.asCQL3Type(),
+                             invalidType.asCQL3Type().toSchemaString(),
```

Comment:
This is going to break some tests checking for exception messages when altering the schema. So far, `CreateTest#testCreateTableWithDurationColumns`. The issue in general is that type errors sometimes try to use the regular `toString` method to use the implicitly frozen syntax, whereas some errors (like non-frozen, non-dropped columns) require `toSchemaString`. I guess it's not that bad to use the schema explicit syntax in all type errors.